mix.exs

```diff
@@ -34,8 +34,9 @@ defmodule EctoSQLite3.MixProject do
   defp deps do
     [
       {:decimal, "~> 1.6 or ~> 2.0"},
-      {:ecto_sql, "~> 3.7"},
-      {:ecto, "~> 3.7"},
+      # TODO: Change back to released version
+      {:ecto_sql, "~> 3.9.0-dev", github: "elixir-ecto/ecto_sql"},
+      {:ecto, "~> 3.9.0-dev", github: "elixir-ecto/ecto"},
```

Comment:
Would we be able to get these back to official packages?

Reply:
@warmwaffles there was no release yet.

Reply:
@warmwaffles The packages are available. I changed the `mix.exs` and updated.